ui/spells/spells.scss

```diff
@@ -115,7 +115,7 @@ $notes-popover-width: 16px;
       font-weight: bold;
       text-align: center;
     }
-    ~:not(.skillname-grid) {
+    ~:not(.skillname-grid, .skillCP-grid) {
```

Comment:
Also fix visibility of CP for innate sphere:
![image](https://user-images.githubusercontent.com/5629800/215367112-21b173a1-2b39-4e36-8cc7-ca569b439a55.png)